scout/server/blueprints/genes/controllers.py

```diff
@@ -26,7 +26,7 @@ def genes(store, query):
     nr_genes_37 = store.nr_genes(build="37")
     nr_genes_38 = store.nr_genes(build="38")
     genes_subset = list(store.all_genes(add_transcripts=False, limit=20))
-    last_inserted_gene = store.hgnc_collection.find({}).sort([("timestamp", -1)]).limit(1)
+    last_inserted_gene = store.hgnc_collection.find({}).sort("_id", -1).limit(1)
```

Comment:
Yes, its basically the same, as the four most significant bytes of the _id is the timestamp!

Reply:
Nope, update time didn't change:

<img width="540" alt="image" src="https://user-images.githubusercontent.com/28093618/167107032-da1b01ae-cdd9-4e49-b66c-fff7d0d2f73e.png">


Perhaps the server timezone?